### PR TITLE
Feature/update collection

### PR DESCRIPTION
**Pull request by IsabellUrban**

### CODE REVIEW

File: components/FormCollection/FormCollection.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  return (
+    <StyledForm onSubmit={handleSubmit} $isFormClosing={isFormClosing}>
+      <StyledFormHeadline>{headline}</StyledFormHeadline>
+
+      <NewCollectionWrapper>
+        <CollectionNameWrapper>
+          <StyledLabel htmlFor="collectionName">Collection Name</StyledLabel>
+          <FormInput
+            name="collectionName"
+            maxlength="23"
```
Comment (Jennifer-Hinrichsen) [inline components/FormCollection/FormCollection.js:44]: Please use camelcase. 
```suggestion
            maxLength="23"
```

File: components/FormCollection/FormCollection.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+      title: collectionName,
+      color: collectionColor,
+    };
+
+    try {
+      await onEditCollection(updatedCollection);
+      event.target.reset();
+      changeActionMode("default");
+    } catch (error) {
+      console.error("Error while editing collection:", error);
```
Comment (Jennifer-Hinrichsen) [inline components/FormCollection/FormCollection.js:31]: Additionally you could add user feedback (e.g., setError or error state for UI rendering).

File: components/FormFlashcard/FormInput.js [hunk 3]
```diff
@@ -5,15 +5,24 @@ export default function FormInput({
   maxlength,
```
Comment (Jennifer-Hinrichsen) [inline components/FormFlashcard/FormInput.js:5]: Please think about naming camelcase.
```suggestion
  maxLength,
```

File: components/FormFlashcard/FormInput.js [hunk 4]
```diff
@@ -5,15 +5,24 @@ export default function FormInput({
...
   currentFlashcard,
+  currentCollection,
 }) {
+  console.log("currentFlashcard", currentFlashcard);
+
   return (
     <StyledInput
       id={name}
       name={name}
       maxLength={maxlength}
```
Comment (Jennifer-Hinrichsen) [inline components/FormFlashcard/FormInput.js:16]: Please think about naming camelcase.
```suggestion
      maxLength={maxLength}
```

**Comment by IsabellUrban** [comment]
Hey, thanks for your comments. I'll add it right away. Since it's only small things, why did you give me a grey tick?

File: components/FormCollection/FormCollection.js (hunk 2 above)
Comment (IsabellUrban) [reply components/FormCollection/FormCollection.js:31]: Hey, I already have error handling for edit Collection in the _App.js.

**Pull request merged**